kpet/utils.py

```diff
@@ -35,6 +36,22 @@ def get_template_path(tree, dbdir):
     return path
 
 
+def get_jinja_template(tree, dbdir):
+    """Get a jinja template instance by tree"""
+    template_dirs = [os.path.join(dbdir, 'trees')]
+    template_dirs.append(os.path.join(dbdir, 'layout'))
+    jinja_env = Environment(
+        loader=FileSystemLoader(template_dirs),
+        autoescape=select_autoescape(
+            enabled_extensions=('html', 'xml'),
```

Comment:
We don't need to enable `html`, do we?
Also, I think the templates should have the standard `.j2` extension, and keep the `.xml` extension to signify what they generate. I.e. have `.xml.j2` on the end. Do you think that would work for you?